README.md

```diff
@@ -53,26 +53,11 @@ podman push $IMG
    ```bash
    oc create ns aws-load-balancer-operator
    ```
-4. Create AWS credentials profile for the operator
-    ```bash
-    cat << EOF > credentials
-    [default]
-    aws_access_key_id=${AWS_ACCESS_KEY_ID}
-    aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
-    EOF
-    
-    oc create secret generic aws-load-balancer-operator -n aws-load-balancer-operator \
-    --from-file=credentials=credentials
-    ```
-   Alternatively use the `CredentialsRequest` resource in the `hack` directory
-   ```bash
-   oc apply -f hack/operator-credentials-request.yaml
-   ```
-5. Deploy the operator
+4. Deploy the operator
     ```bash
     make deploy
     ```
-6. The previous step deploys the conversion webhook, which requires TLS verification on the webhook client side. The
+5. The previous step deploys the conversion webhook, which requires TLS verification on the webhook client side. The
    manifests deployed through the `make deploy` command do not contain a valid certificate and key. You must provision a valid certificate and key through other tools.     
    If you run on OpenShift, you can use a convenience script, `hack/add-serving-cert.sh`, to enable [the service serving certificate feature](https://docs.openshift.com/container-platform/4.12/security/certificates/service-serving-certificate.html). 
```

Comment:
Same answer as for https://github.com/openshift/aws-load-balancer-operator/pull/109/files#r1337623230.
However this one can be updated to 4.13 as the docs don't seem to change since 4.12.